### DISCUSSION TITLE
How to access response status after calling fetcher.submit()?

### DISCUSSION THREAD

**Discussion by maciej-adamus** [remix-run/remix#4425] [Q&A] [answered]
How do I access the response status after calling `fetcher.submit()`? The `fetcher.data` seems to contain only the response, without status.

**Comment by sergiodxa** [comment, marked as answer]
There's no way, you'll have to add the status to the data

**Reply by kiliman** [reply]
Yeah, I wished fetchers exposed the underlying Response.

**Reply by sergiodxa** [reply]
Yeah a `fetcher.response` would be useful

**Reply by danielweinmann** [reply]
Just to add to the discussion, a `fetcher.request` would also be handy for me. Sometimes I want to know to which href my latest `fetcher.load` was sent, and right now I'm having to solve it with state.
